indico/modules/users/controllers.py

```diff
@@ -402,7 +402,7 @@ def _process(self):
             external = form_data.pop('external')
             form_data = {k: v for (k, v) in form_data.iteritems() if v and v.strip()}
             matches = search_users(exact=exact, include_deleted=include_deleted, include_pending=include_pending,
-                                   external=external, allow_system_user=True, **form_data)
+                                   include_blocked=True, external=external, allow_system_user=True, **form_data)
```

Comment:
In the admin search, I'm just including the blocked users by default because we already have way too many options and a label will be shown anyway. Nonetheless, I will leave this thread for change requests if anyone disagrees.